src/blocks/attendance-list/render.php

```diff
@@ -2,9 +2,9 @@
 /**
  * Render Attendance List block.
  *
- * @package GatherPress
+ * @package    GatherPress
  * @subpackage Core
- * @since 1.0.0
+ * @since      1.0.0
```

Comment:
revert all these indent changes in what appears to be only the `render.php` files, lining up comments in these blocks isn't part of the standard.